contrib/post-receive

```diff
@@ -12,9 +12,16 @@ echo "\n==> Activating virtualenv…\n"
 . /opt/jarbas.venv/bin/activate
 
 echo "\n==> Installing NodeJS packages…\n"
+if [ -d node_modules/ ]; then
+    rm -rf node_modules/
+fi
+if [ -d elm-stuff/ ]; then
+    rm -rf elm-stuff/
+fi
 yarn install
 
 echo "\n==> Installing Python packages…\n"
+pip install -U pip
 pip install -r requirements.txt
```

Comment:
I like the `-U` because sometimes simply running `pip install -r requirements.txt` did not update some libs I had in a previous version even when requirements had an higher version.